code/modules/food_and_drinks/food/candy.dm

```diff
@@ -134,14 +128,15 @@
 	trash = /obj/item/trash/candy
 	filling_color = "#7D5F46"
 	bitesize = 3
-	list_reagents = list("nutriment" = 2, "chocolate" = 5)
+	junkiness = 25
```

Comment:
Honestly, all candy products should probably have junkiness, not just candy bars. Also, the chef-made candy bars (Asteroid Crunch Bar, Tool-erone, Toxins Test Bar, Yum-Baton, and Malper Bar) are subtypes of this `/obj/item/weapon/reagent_container/food/snacks/candy/candybar`, so they will inherit this junkiness in case you weren't intending that.